dts/arm/nxp/nxp_lpc55S6x_ns.dtsi

```diff
@@ -7,7 +7,7 @@
 / {
 	soc {
 		sram: sram@400000 {
```

Comment:
The node name should also get an extra zero.

Reply:
fixed